l3build-help.lua

```diff
@@ -82,10 +84,13 @@ function help()
       end
     end
   end
-  print("")
-  print("Full manual available via 'texdoc l3build'.")
-  print("")
-  print("Repository  : https://github.com/latex3/l3build")
-  print("Bug tracker : https://github.com/latex3/l3build/issues")
-  print("Copyright (C) 2014-2020 The LaTeX3 Project")
+  print([[
+Full manual available via 'texdoc l3build'.
+
+Repository  : https://github.com/latex3/l3build
+Bug tracker : https://github.com/latex3/l3build/issues
+Copyright (C) 2014-2020 The LaTeX3 Project
```

Comment:
We can't change this line without also adjusting the copyright-update code in the `build.lua` file for `l3build` itself (so tagging works properly). Same for the other copyight-containing lines.